### ISSUE TITLE
add option to hide drop location

### ISSUE THREAD

**Issue by bleything** [fuchsberger/lotd#17]
What do you think about adding the ability to hide the location where the item drops? Sometimes it's nice to have to hunt a bit for an item...

**Comment by fuchsberger** [comment]
good point and technically not to hard to implement. Ill see when I get around to do it. thanks for the suggestion!
